packages/uswds-core/src/styles/tokens/color/high-contrast-mode-colors.scss

```diff
@@ -0,0 +1,19 @@
+$high-contrast-system-colors: (
```

Comment:
In stable branch these were added to `_system-tokens.scss`. I've created a separate file for these and included them in `shortcodes-color-all`.

https://github.com/uswds/uswds/blob/88b648e069034105607c0290b2e41dc13a97f507/src/stylesheets/core/_system-tokens.scss#L1701-L1719